pep8 1.5.4 fails on html5lib

```
$ bash flake8-run.sh
html5lib/sanitizer.py:248:24: E713 test for membership should be 'not in'
html5lib/treebuilders/dom.py:161:24: E713 test for membership should be 'not in'
html5lib/tests/test_tokenizer.py:71:12: E713 test for membership should be 'not in'
html5lib/tests/test_tokenizer.py:72:34: E713 test for membership should be 'not in'
html5lib/tests/test_tokenizer.py:115:40: E128 continuation line under-indented for visual indent
html5lib/serializer/htmlserializer.py:38:16: E713 test for membership should be 'not in'
html5lib/serializer/htmlserializer.py:294:20: E713 test for membership should be 'not in'
```

--- a/html5lib/html5parser.py
+++ b/html5lib/html5parser.py
@@ -1216,8 +1216,7 @@ def getPhases(debug):
             attributes["name"] = "isindex"
             self.processStartTag(impliedTagToken("input", "StartTag",
                                                  attributes=attributes,
-                                                 selfClosing=
-                                                 token["selfClosing"]))
+                                                 selfClosing=token["selfClosing"]))
             self.processEndTag(impliedTagToken("label"))
             self.processStartTag(impliedTagToken("hr", "StartTag"))
             self.processEndTag(impliedTagToken("form"))

--- a/html5lib/sanitizer.py
+++ b/html5lib/sanitizer.py
@@ -245,7 +245,7 @@ class HTMLSanitizerMixin(object):
             elif prop.split('-')[0].lower() in ['background', 'border', 'margin',
                                                 'padding']:
                 for keyword in value.split():
-                    if not keyword in self.acceptable_css_keywords and \
+                    if keyword not in self.acceptable_css_keywords and \
                             not re.match("^(#[0-9a-f]+|rgb\(\d+%?,\d*%?,?\d*%?\)?|\d{0,2}\.?\d{0,2}(cm|em|ex|in|mm|pc|pt|px|%|,|\))?)$", keyword):
                         break
                 else:

--- a/html5lib/serializer/htmlserializer.py
+++ b/html5lib/serializer/htmlserializer.py
@@ -35,7 +35,7 @@ else:
                 v = utils.surrogatePairToCodepoint(v)
             else:
                 v = ord(v)
-            if not v in encode_entity_map or k.islower():
+            if v not in encode_entity_map or k.islower():
                 # prefer &lt; over &LT; and similarly for &amp;, &gt;, etc.
                 encode_entity_map[v] = k
 
@@ -291,7 +291,7 @@ class HTMLSerializer(object):
             elif type == "Entity":
                 name = token["name"]
                 key = name + ";"
-                if not key in entities:
+                if key not in entities:
                     self.serializeError(_("Entity %s not recognized" % name))
                 if self.resolve_entities and key not in xmlEntities:
                     data = entities[key]

--- a/html5lib/tests/test_tokenizer.py
+++ b/html5lib/tests/test_tokenizer.py
@@ -68,8 +68,8 @@ class TokenizerTestParser(object):
 def concatenateCharacterTokens(tokens):
     outputTokens = []
     for token in tokens:
-        if not "ParseError" in token and token[0] == "Character":
-            if (outputTokens and not "ParseError" in outputTokens[-1] and
+        if "ParseError" not in token and token[0] == "Character":
+            if (outputTokens and "ParseError" not in outputTokens[-1] and
                     outputTokens[-1][0] == "Character"):
                 outputTokens[-1][1] += token[1]
             else:
@@ -112,7 +112,7 @@ def tokensMatch(expectedTokens, receivedTokens, ignoreErrorOrder,
         # Sort the tokens into two groups; non-parse errors and parse errors
         tokens = {"expected": [[], []], "received": [[], []]}
         for tokenType, tokenList in zip(list(tokens.keys()),
-                                       (expectedTokens, receivedTokens)):
+                                        (expectedTokens, receivedTokens)):
             for token in tokenList:
                 if token != "ParseError":
                     tokens[tokenType][0].append(token)

--- a/html5lib/treebuilders/dom.py
+++ b/html5lib/treebuilders/dom.py
@@ -158,7 +158,7 @@ def getDomBuilder(DomImplementation):
             else:
                 # HACK: allow text nodes as children of the document node
                 if hasattr(self.dom, '_child_node_types'):
-                    if not Node.TEXT_NODE in self.dom._child_node_types:
+                    if Node.TEXT_NODE not in self.dom._child_node_types:
                         self.dom._child_node_types = list(self.dom._child_node_types)
                         self.dom._child_node_types.append(Node.TEXT_NODE)
                 self.dom.appendChild(self.dom.createTextNode(data))

